libs/application/templates/transport-authority/exemption-for-transportation/src/utils/helperFunctions/format.ts

```
@@ -6,5 +6,13 @@ export const formatDate = (date: Date): string =>
 export const formatDateStr = (dateStr: string | undefined): string =>
   dateStr ? formatDate(new Date(dateStr)) : ''
 
-export const formatNumber = (num?: number) =>
-  num !== undefined && num !== null ? num.toLocaleString('de-DE') : ''
+export const formatNumber = (value?: number | string) => {
+  const num = Number(value)
+  return isFinite(num) ? num.toLocaleString('de-DE') : ''
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)